petstore/clients/javascript/samples/typescript/src/petOperations.ts

```diff
@@ -0,0 +1,28 @@
+import { PetStoreClient } from '@unbranded/petstore';
+
+const client = new PetStoreClient("http://localhost:5118", {
```

Comment:
is this a fixed endpoint that we all use? can we use environment variables?

Reply:
updated